packages/superset-ui-legacy-plugin-chart-force-directed/src/ForceDirected.js

```diff
@@ -38,11 +38,18 @@ const propTypes = {
 /* Modified from http://bl.ocks.org/d3noob/5141278 */
 function ForceDirected(element, props) {
   const { data, width, height, linkLength = 200, charge = -500 } = props;
+
+  let w = window.innerWidth;
```

Comment:
The chart does not always have `width` and `height` being 100% of the window `width` and `height`.

Reply:
Thanks for the input. I commited a version where width/height are inherited.